src/operator/src/req_convert/insert/row_to_region.rs

```diff
@@ -49,10 +50,36 @@ impl<'a> RowToRegion<'a> {
         let mut region_request = Vec::with_capacity(requests.inserts.len());
         let mut instant_request = Vec::with_capacity(requests.inserts.len());
         for request in requests.inserts {
+            let Some(rows) = request.rows else { continue };
+
             let table_id = self.get_table_id(&request.table_name)?;
-            let requests = Partitioner::new(self.partition_manager)
-                .partition_insert_requests(table_id, request.rows.unwrap_or_default())
-                .await?;
+            let table_route = self
+                .partition_manager
+                .find_physical_table_route(table_id)
```

Comment:
I am concerned this will cause performance loss with multi-partition, as `find_physical_table_route` will be called again internally.

Reply:
use already retrived table info now, no extra perf penalty